docs/configuration.md

```diff
@@ -1,831 +1 @@
-# Configuration
```

Comment:
Why are you deleting this file?

Reply:
No remote idea. I guess this is the combination of macos filesystem issues and `git add *`.